app/controllers/application.js

```diff
@@ -62,6 +65,12 @@ export default Controller.extend(
 			});
 
 			this._super();
+
+			track({
+				action: trackActions.open,
+				category: 'connection-type',
+				label: `mobile-${this.get('connectionType')}`
```

Comment:
consider sth like `Mobile: ${this.get('connectionType')}`

Reply:
Can you be more specific with this one? :)

Reply:
I wanted to get rid of "-", but others convinced me to remove "mobile" completely, that's why you can skip this comment :D